programmers/20210503/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+    for(var i = 0; i < seoul.length; i++){
+        if (seoul[i] == "Kim"){
+            answer = i;
+        }
+    }
+    var answer = "김서방은 " + answer + "에 있다";
+    return answer;
+}
```

Comment:
지금 짜신 방식도 좋지만 자바스크립트의 내장 함수를 이용하시면 더 편하게 코드를 짜실 수 있습니다. 
indexOf는 배열 안에서 주어진 원소의 index위치를 찾습니다.
```suggestion
function solution(seoul) {
   return `김서방은 ${seoul.indexOf("Kim")}에 있다`;
}
```